managers/models.py

```diff
@@ -8,8 +12,34 @@ class Project(models.Model):
     start_date = models.DateField()
     stop_date = models.DateField(null=True, blank=True)
     terminated = models.BooleanField(default=False)
-    managers = models.ManyToManyField(CustomUser, related_name="managers")
-    members = models.ManyToManyField(CustomUser, related_name="members")
+    managers = models.ManyToManyField(CustomUser, related_name="managed_projects")
```

Comment:
I'd suggest to change this name to `manager_projects`.

Reply:
Done